test/jpypetest/test_startup.py

```diff
@@ -81,6 +79,16 @@ def testClasspathArgListEmpty(self):
     def testClasspathArgDef(self):
         runStartJVMTest('-Djava.class.path=%s' % cp, convertStrings=False)
 
+    def testClasspathArgPath(self):
+        runStartJVMTest(classpath=Path(cp), convertStrings=False)
+
+    def testClasspathArgPathList(self):
+        runStartJVMTest(classpath=[Path(cp)], convertStrings=False)
+
+    def testClasspathArgGlob(self):
+        jpype.startJVM(classpath=os.path.join(cp, '..', 'jar', 'mrjar*'))
```

Comment:
Don't know if I'm allowed to rely on this JAR for testing here...

Reply:
looks unknown to me

Reply:
It is part of the test suite (hence the test passing in CI), but indeed it is not ideal to have such a weak connection between this test and the underlying test functionality.

Not entirely sure how to proceed though, without making this test much more complex.

Perhaps @Thrameos has an idea for this one?